.gitignore

```diff
@@ -13,3 +13,8 @@
 
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
+
+# Cache
+packages/android
+packages/ios
```

Comment:
Can you remove this? (see Makefile comments)

Reply:
Done.